src/docs/guides/symfony.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Deploy a Symfony App
+---
+
+[Symfony](https://symfony.com) is a PHP web framework composed of a set of decoupled and reusable components all working together in harmony to create websites and web applications.
+
+This guide covers how to deploy a Symfony app to Railway in three ways:
+
+1. [One-click deploy from a template](#one-click-deploy-from-a-template).
+2. [From a GitHub repository](#deploy-from-a-github-repo).
+3. [Using the CLI](#deploy-from-the-cli).
+
+Now, let's create a Symfony app!
+
+## Create a Symfony App
+
+**Note:** If you already have a Symfony app locally or on GitHub, you can skip this step and go straight to the [Deploy Symfony App to Railway](#deploy-the-symfony-app-to-railway).
+
+To create a new Symfony app, ensure that you have [Composer](https://getcomposer.org/download/), [PHP](https://www.php.net/manual/en/install.php) and [Symfony](https://symfony.com/download) installed on your machine.
+
+Run the following command in your terminal to create a new Symfony app:
+
+```bash
+symfony new --webapp apphelloworld
+```
+
+A new Symfony app will be provisioned for you in the `apphelloworld` directory.
+
+### Run the Symfony App locally
+
+To start your app, run:
+
+```bash
+symfony server:start
+```
+
+Once the app is running, open your browser and navigate to `http://localhost:8000` to view it in action.
+
+## Deploy the Symfony App to Railway
+
+Railway offers multiple ways to deploy your Symfony app, depending on your setup and preference. 
+
+## One-Click Deploy from a Template
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/4tnH_D)
```

Comment:
Yeah, I should make it capitalized. Note to self... 🗒️ 

I made APP_DEBUG set to true because I was testing something, so yeah I should disable that.

> I don't think NIXPACKS_PHP_ROOT_DIR would be used on the worker or cron service, and in that case, it should be removed.
You are absolutely right. I just copied all the env variables to each service...but in retrospect, these 2 don't actually need it.